SwiftLint/swiftlint.yml

```diff
@@ -198,7 +203,7 @@ custom_rules:
     severity: warning
   missing_brackets:
     name: "Missing brackets"
-    regex: '(^(\t| )*(if|\} else if) (((?!(let|var|\())).)*\{)'
+    regex: '(^(\t| )*(if|\} else if) (((?!(case|let|var|\())).)*\{)'
```

Comment:
@proth Please create another PR for this update as it is a different subject than the new rule.